dspace/config/item-submission.xml

```diff
@@ -267,6 +274,10 @@
             <!-- To include the 'Show Identifiers' step, uncomment the line below. Make sure
                  that identifiers.cfg and identifier-service.xml are properly configured. -->
             <!--<step id="identifiers"/>-->
+            <!-- To include the 'Duplicate Detection' step, uncomment the line below. Make sure
+                 that duplicate-detection.cfg is properly configured and items have been reindexed since
+                 enabling the feature -->
+            <!--<step id="duplicates"/>-->
```

Comment:
Tiny thing...it might be more "useful" to move this step to *after* the metadata entry steps below.  I found in testing that I didn't notice the duplicate step at first because I had to scroll back to the top of the submission form to see where it appeared.  This is an optional suggestion though as sites can move this wherever they wish.

Reply:
Good call, I agree - especially since a new submission needs metadata before a duplicate match makes sense. Users will customise this to fit their purpose anyway. I have made this change.